using-mobile-extensions/adobe-edge-consent/edge-consent-api-reference.md

```diff
@@ -0,0 +1,290 @@
+# Consent Collection API reference
+
+## extensionVersion <a id="extensionversion"></a>
+
+Returns the version of the client-side Edge Consent extension.
+
+{% tabs %}
+{% tab title="Android" %}
+### Java
+
+#### Syntax
+
+```java
+public static String extensionVersion();
+```
+
+#### Example
+
+```java
+Consent.extensionVersion();
+```
+{% endtab %}
+
+{% tab title="iOS" %}
+### Swift
+
+#### Syntax
+
+```swift
+public static let extensionVersion
+```
+
+#### Example
+
+```swift
+Consent.extensionVersion
+```
+
+### Objective-C
+
+#### Syntax
+
+```swift
+public static let extensionVersion
+```
+
+#### Example
+
+```objective-c
+[AEPMobileEdgeConsent extensionVersion];
+```
+{% endtab %}
+{% endtabs %}
+
+## registerExtension <a id="registerextension"></a>
```

Comment:
let's sort the APIs alphabetically as we do with other extensions

Reply:
I think makes no sense that registerExtension API is at the bottom of the docs. It's literally the first API you need to use...

Reply:
And if we sort these alphabetically registerExtension will come between get and update consents

1. extension version
2. get consents
3. register extension
4. update consents

I understand we've sorted other APIs alphabetically, but I think it makes much more sense to sort them in a manner that makes sense to someone who is using the APIs.